File: CS/Form1.cs
using System;
using System.Windows.Forms;

using DevExpress.XtraReports.UserDesigner;
// ...

namespace EUD_ExecCommandCS {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            // Create an End-User Designer form instance.
            XRDesignFormEx designForm = new XRDesignFormEx();

            // Handle its Load event.
            designForm.Load += new EventHandler(designForm_Load);

            // Show this form modally.
            designForm.ShowDialog();
        }

        private void designForm_Load(object sender, EventArgs e) {
            // Create a new report and run the Report Wizard every time a form is loaded.
            ((XRDesignFormEx)sender).DesignPanel.ExecCommand(ReportCommand.NewReportWizard);
        }
    }
}

File: VB/Form1.vb
Imports DevExpress.XtraReports.UserDesigner
' ...

Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim designForm As New XRDesignFormEx()
        AddHandler designForm.Load, AddressOf designForm_Load
        designForm.ShowDialog()
    End Sub

    Private Sub designForm_Load(ByVal sender As Object, ByVal e As EventArgs)
        CType(sender, XRDesignFormEx).DesignPanel.ExecCommand(ReportCommand.NewReportWizard)
    End Sub
End Class
